spec/core/QueueRunnerSpec.js

```diff
@@ -234,6 +234,52 @@ describe("QueueRunner", function() {
       queueRunner.execute();
       expect(doneReturn).toBe(null);
     });
+
+    it("supports promises (and thenables) returned by queued functions", function(done) {
+      var queueableFn1 = { fn: function() {
+          return Promise.resolve();
```

Comment:
Jasmine's tests need to run in IE 8+ and PhantomJS (as well as Edge, Chrome, Firefox, and Node.js) so we can't use bare `Promise` directly in specs for browsers that don't support them.